pelias/README.md

```diff
@@ -0,0 +1,168 @@
+# Setup guide for Pelias geocoding service.
+
+This guide explains how we set up the geocoding service for https://best.osoc.be.
```

Comment:
Even though, GitHub make links clickable, try to always put your links between `<>` (like this `<https://best.osoc.be>`) just like you did line 26.
This is only needed when you display the whole URL!

Reply:
This should be fixed now!